Angular/package.json

```diff
@@ -14,24 +14,24 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^15.1.0",
-    "@angular/common": "^15.1.0",
-    "@angular/compiler": "^15.1.0",
-    "@angular/core": "^15.1.0",
-    "@angular/forms": "^15.1.0",
-    "@angular/platform-browser": "^15.1.0",
-    "@angular/platform-browser-dynamic": "^15.1.0",
-    "@angular/router": "^15.1.0",
-    "devextreme": "^23.1.3",
-    "devextreme-angular": "^23.1.3",
+    "@angular/animations": "^16.1.0",
```

Comment:
16 looks quite old as well, maybe Angular 18?

Reply:
+ 1 to v18

Reply:
Angular update to the 18th version 🙌